doc/whats_new/v1.5.rst

```diff
@@ -171,6 +171,14 @@ Changelog
 - |Enhancement| :term:`CV splitters <CV splitter>` that ignores the group parameter now
   raises a warning when groups are passed in to :term:`split`. :pr:`28210` by
 
+- |Feature| :class:`model_selection.GroupKFold` now has the ability to shuffle groups
+  into different folds when `shuffle=True`.
+  :pr:`28519` by :user:`Zachary Vealey <zvealey>`.
+
+- |Feature| :class:`model_selection.RepeatedGroupKFold` has been created for repeating
```

Comment:
Could you do a separate PR for this one. Basically, we prefer to merge a single feature by pull-request. It is more atomic and ease the review process.